crates/typst/src/geom/path.rs

```diff
@@ -51,4 +51,28 @@ impl Path {
     pub fn close_path(&mut self) {
         self.0.push(PathItem::ClosePath);
     }
+
+    /// Computes the size of the path.
+    pub fn size(&self) -> Size {
```

Comment:
`size` doesn't feel super well-defined. maybe `bounds_size` / `bbox_size` or something? just `bounds` / `bbox` might be confusing because it doesn't return a rect, only a size.